lib/IDE/SyntaxModel.cpp

```diff
@@ -1318,6 +1315,7 @@ bool ModelASTWalker::findUrlStartingLoc(StringRef Text,
       return true;
     }
   }
+#endif
   return false;
```

Comment:
Wouldn't this cause equivalent unreachable code warnings for `return false`?

Could you do `#ifdef ... #else ... #endif` instead?